.github/workflows/coin-integration.yml

```diff
@@ -0,0 +1,42 @@
+name: coin-integration-alpha
+on:
+  push:
+    branches:
+      - 'int-*'
+jobs:
+  prebuild-for-platform:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, macos-latest, windows-2016]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+          registry-url: https://registry.npmjs.org
+          always-auth: true
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '2.7.x'
+      - name: install
+        run: yarn --frozen-lockfile --ignore-scripts
+      - name: fetch libcore
+        run: yarn preinstall
+      - name: config gcc
+        if: runner.os == 'Linux'
+        run: sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 700 --slave /usr/bin/g++ g++ /usr/bin/g++-7
+      - name: prebuild for electron 8.1
```

Comment:
Can you update 8.1 to 9 ? Check on master we merged a new version

Reply:
Will do, no idea why it has not updated yet

Reply:
Not staging the changes is one answer....